src/Table/index.stories.tsx

```diff
@@ -41,22 +41,31 @@ const data: Array<Person> = [
   },
 ];
 
-const FilterDropdown: ColumnProps<Person>['filterDropdown'] = () => (
-  <div>test</div>
-);
-
 const columns: Array<ColumnProps<Person>> = [
   {
     title: 'Name',
     dataIndex: 'name',
-    filterDropdown: FilterDropdown,
```

Comment:
Warum entfernt?

Reply:
The implementation was removed here. Thus, I do not see a benefit in keeping the visual representation in storybook.

Reply:
The workaround for an issue with antd was removed. The feature still exists, is useful and can be explored:

![image](https://user-images.githubusercontent.com/12158000/127634266-ef6173fb-3f25-4090-9fb6-20a914e31a86.png)